tests/test_trainer.py

```diff
@@ -155,6 +157,17 @@ def test_plm(self):
 
 @require_torch
 class TrainerIntegrationTest(unittest.TestCase):
+    class SampleIterableDataset(IterableDataset):
+        def __init__(self, file_path):
+            self.file_path = file_path
+
+        def parse_file(self):
+            f = open(self.file_path, "r")
+            return f.readlines()
+
+        def __iter__(self):
+            return iter(self.parse_file())
+
     def test_trainer_eval_mrpc(self):
         MODEL_ID = "bert-base-cased-finetuned-mrpc"
```

Comment:
you could change this too.

Reply:
Which is the tiny version for mrpc task?

Reply:
just leave, it. Not fair of me to ask you to get to the bottom and they check the loss below. Sorry.